### DISCUSSION TITLE
"Use this package" to reference `swift package` commands

### DISCUSSION THREAD

**Discussion by MahdiBM** [SwiftPackageIndex/SwiftPackageIndex-Server#3188] [Ideas]
We have a new `add-dependency` command in Swift 6:

```bash
swift package add-dependency https://github.com/DiscordBM/DiscordBM --from 1.12.0
```

SPI's "Use this package" view should include this, with a small note that it requires Swift 6 CLI.

<kbd> <img width="664" alt="Screenshot 2024-07-05 at 9 28 25 AM" src="https://github.com/SwiftPackageIndex/SwiftPackageIndex-Server/assets/54685446/e8546c73-95fb-40f9-8dbd-9af620002421"> </kbd>

crates.io also references these commands, which reminded me that we recently got them in Swift too, so SPI can also do that:

<kbd> <img width="268" alt="Screenshot 2024-07-05 at 9 32 12 AM" src="https://github.com/SwiftPackageIndex/SwiftPackageIndex-Server/assets/54685446/887d9d90-0ff5-4813-96dd-a8af9295df7d"> </kbd>

**Comment by daveverwer** [comment]
Thanks for this @MahdiBM, I knew about the new command but hadn't thought properly about how best to integrate it here. I'll promote this to an issue so we make sure that happens.

**Comment by daveverwer** [comment]
Tracked in https://github.com/SwiftPackageIndex/SwiftPackageIndex-Server/issues/3189
